bedrock/settings/base.py

```diff
@@ -236,6 +236,7 @@ def data_path(*args):
     "banners/firefox-app-store",
     "banners/fundraising",
     "banners/vpn-cyber-security-month",
```

Comment:
Question: can we remove the old banner and related files now this new one is replacing it?